### PR TITLE
Introduce thamos check command

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/thamos/issues/630

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thamos/config.py [hunk 1]
```diff
@@ -41,6 +42,77 @@
 _LOGGER = logging.getLogger(__name__)
 _THAMOS_DISABLE_TLS_WARNING = bool(int(os.getenv("THAMOS_DISABLE_TLS_WARNING", 0)))
 
+# The schema is enforcing all the options. This will make sure the right version of Thamos is
+# installed and no configuration options are silently ignored.
+_CONFIG_SCHEMA = {
```
Comment (goern) [inline thamos/config.py:47]: is the config schema something we should have as a separate file?

Comment (fridex) [reply thamos/config.py:47]: Is there any functionality that would require it?

Comment (fridex) [reply thamos/config.py:47]: I didn't want to expose this outside of the module. Users can use library functions for validating the schema. If you wish to have it separated, let me know. I would like to include this functionality in the next s2i release so we guarantee users use correct configuration options during the s2i build process.

Comment (goern) [reply thamos/config.py:47]: It could come in handy, but is note required..

**Pull request merged**